u8timeseries/custom_logging.py

```diff
@@ -16,10 +16,10 @@ def get_logger(name):
     """
     logger = logging.getLogger(name)
     logger.setLevel(logging.INFO)
-    stderr_handler = logging.StreamHandler()
+    handler = logging.StreamHandler()
```

Comment:
Is there a way to check what was logged, but not print it? For instance, keep every log in memory and then check if some message is in memory?

Reply:
I think the behaviour you're describing is achieved by the LogCapture class from testfixtures. This class can catch logs without printing them and lets us check for the presence of a specific log. But this only works for the root logger. That means that if we create our own logger with an additional handler, the logs from that handler do not get prevented from printing (note that every logger by default propagates its logs to the root logger as well). This is why, before testing log messages using LogCapture, I replace the handler with the NullHandler (although I could have just removed the handler as well).
I hope all of this makes sense.